frontend/src/scenes/settings/environment/ManagedReverseProxy.tsx

```diff
@@ -144,7 +145,7 @@ export function ManagedReverseProxy(): JSX.Element {
                 {formState === 'collapsed' ? (
                     maxRecordsReached ? (
                         <LemonBanner type="info">
-                            There is a maximum of {MAX_PROXY_RECORDS} records allowed per organization
+                            There is a maximum of {featureAvailableOnOrg?.limit || 0} records allowed per organization.
```

Comment:
Not sure what the plans are with regards to pricing and possible upgrades for more proxies but the copy here could include a CTA for this (if/once we have it).

Reply:
It's limiting to just teams / enterprise right now, some users have customer overrides for it. The pay gate wrapper handles the upgrade logic.